dedibox-console/classic-hosting/how-to/access-logcenter.mdx

```diff
@@ -26,17 +26,17 @@ The logcenter allows you to visualize real-time error logs of your hosting. That
 
 1. From your Dedibox console, click **Hosting**. A list of your webhosting accounts displays.
 2. Click **Manage** next to the hosting you want to configure. The webhosting overview page displays.
-3. Click **Statistics and logs** in the sidemenu. The statistics and logs overview page displays.
+3. Click **Statistics and logs** in the side menu. The statistics and logs overview page displays.
 4. Go to the **Logs** section and click **Define password**. If you want to reset your password, click **Reset password**.
-    <Lightbox src="scaleway_logcenter_define_password.webp" alt="" />
```

Comment:
does it work without the `alt=""`? iirc the build failed without it

Reply:
I've removed it from previous PRs, and it worked. But I can leave this on, just in case.